### PR TITLE
Added README files for Presentation interactive sample browser

### PR DESCRIPTION
**Pull request by Suriya-Balamurugan**
Added README files for Presentation interactive sample browser.

### CODE REVIEW

File: PPTX-to-Image-conversion/Convert-PowerPoint-slide-to-Image/.NET/Convert-PowerPoint-slide-to-Image/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+//Load or open an PowerPoint Presentation.
+using FileStream inputStream = new(Path.GetFullPath(@"Data/Template.pptx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
+//Open an existing PowerPoint presentation.
+using IPresentation pptxDoc = Presentation.Open(inputStream);
+//Initialize the PresentationRenderer to perform image conversion.
+pptxDoc.PresentationRenderer = new PresentationRenderer();
+//Convert PowerPoint slide to image as stream.
+using Stream stream = pptxDoc.Slides[0].ConvertToImage(ExportImageFormat.Jpeg);
+//Reset the stream position.
+stream.Position = 0;
```
Comment (MohanaselvamJothi) [inline PPTX-to-Image-conversion/Convert-PowerPoint-slide-to-Image/.NET/Convert-PowerPoint-slide-to-Image/README.md:31]: Check and remove this line in sample itself, if not need

File: PPTX-to-PDF-conversion/Convert-PowerPoint-into-accessible-PDF/.NET/Convert-PowerPoint-into-accessible-PDF/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+//Instantiation of the PresentationToPdfConverterSettings.
+PresentationToPdfConverterSettings pdfConverterSettings = new PresentationToPdfConverterSettings();
+//Enable a flag to preserve structured document tags in the converted PDF document.               
+pdfConverterSettings.AutoTag = true;
+//Convert the PowerPoint document to a PDF document.
+using PdfDocument pdfDocument = PresentationToPdfConverter.Convert(pptxDoc, pdfConverterSettings);
+//Save the converted PDF document to the fileStream.
+using FileStream fileStreamOutput = File.Create(Path.GetFullPath("Output/PPTXToPDF.pdf"));
+pdfDocument.Save(fileStreamOutput);
+fileStreamOutput.Position = 0;
```
Comment (MohanaselvamJothi) [inline PPTX-to-PDF-conversion/Convert-PowerPoint-into-accessible-PDF/.NET/Convert-PowerPoint-into-accessible-PDF/README.md:36]: Check and remove this line if not need

File: PPTX-to-PDF-conversion/Convert-PowerPoint-presentation-to-PDF/.NET/Convert-PowerPoint-presentation-to-PDF/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+using FileStream inputStream = new(Path.GetFullPath(@"Data/Template.pptx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
+//Open an existing PowerPoint presentation.
+using IPresentation pptxDoc = Presentation.Open(inputStream);
+//Create the MemoryStream to save the converted PDF.
+using MemoryStream pdfStream = new();
+//Convert the PowerPoint document to PDF document.
+using PdfDocument pdfDocument = PresentationToPdfConverter.Convert(pptxDoc);
+//Save the converted PDF document to MemoryStream.
+pdfDocument.Save(pdfStream);
+pdfStream.Position = 0;
```
Comment (MohanaselvamJothi) [inline PPTX-to-PDF-conversion/Convert-PowerPoint-presentation-to-PDF/.NET/Convert-PowerPoint-presentation-to-PDF/README.md:33]: Check and remove if not need

File: Slides/Merge-PowerPoint-slide/.NET/Merge-PowerPoint-slide/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Merge PowerPoint Presentations using C#
```
Comment (MohanaselvamJothi) [inline Slides/Merge-PowerPoint-slide/.NET/Merge-PowerPoint-slide/README.md:1]: Create readme file for new sample

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: PPTX-to-Image-conversion/Convert-PowerPoint-slide-to-Image/.NET/Convert-PowerPoint-slide-to-Image/README.md (hunk 1 above)
Comment (Suriya-Balamurugan) [reply PPTX-to-Image-conversion/Convert-PowerPoint-slide-to-Image/.NET/Convert-PowerPoint-slide-to-Image/README.md]: Addressed

File: PPTX-to-PDF-conversion/Convert-PowerPoint-presentation-to-PDF/.NET/Convert-PowerPoint-presentation-to-PDF/README.md (hunk 3 above)
Comment (Suriya-Balamurugan) [reply PPTX-to-PDF-conversion/Convert-PowerPoint-presentation-to-PDF/.NET/Convert-PowerPoint-presentation-to-PDF/README.md]: Addressed

File: Slides/Merge-PowerPoint-slide/.NET/Merge-PowerPoint-slide/README.md (hunk 4 above)
Comment (Suriya-Balamurugan) [reply Slides/Merge-PowerPoint-slide/.NET/Merge-PowerPoint-slide/README.md]: Addressed

File: PPTX-to-PDF-conversion/Convert-PowerPoint-into-accessible-PDF/.NET/Convert-PowerPoint-into-accessible-PDF/README.md (hunk 2 above)
Comment (Suriya-Balamurugan) [reply PPTX-to-PDF-conversion/Convert-PowerPoint-into-accessible-PDF/.NET/Convert-PowerPoint-into-accessible-PDF/README.md]: Addressed

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**